sdks/python/apache_beam/examples/kafkataxi/README.md

```diff
@@ -0,0 +1,164 @@
+[kafka.py](https://github.com/apache/beam/blob/master/sdks/python/apache_beam/io/kafka.py).
+Transforms are implemented in Java and are available
+[here](https://github.com/apache/beam/blob/master/sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java).
+
+## Prerequisites
+
+Install Java in your system and make sure that `java` command is available in 
+the environment.
+
+```sh
+>java --version
+> <Should print information regarding the installed Java version>
+```
+
+## Setup the Kafka cluster
+
+This example requires users to setup a Kafka cluster that the Beam runner
+executing the pipeline has access to. There are few options.
+
+* For local runners that execute the pipelines in a single computer (for 
+example, portable DirectRunner or Spark/Flink runners in local mode), you can 
+setup a local Kafka cluster running in the same computer.
+* For Dataflow or portable Spark/Flink in distributed mode, you can setup a Kafka 
+cluster in GCE. See 
+[here](https://github.com/GoogleCloudPlatform/java-docs-samples/tree/master/dataflow/flex-templates/kafka_to_bigquery) 
+for step by step instructions for this.
+
+Let's assume that that IP address of the node running the Kafka cluster to be 
+`KAFKA_ADDRESS` and the port to be `9092`.
+
+```sh
+> export BOOTSTRAP_SERVER=KAFKA_ADDRESS:9092
+```
+
+## Running the example on latest released Beam version
+
+Perform Beam runner specific setup. Note that cross-language transforms require 
+portable implementations of Spark/Flink/Direct runners. Dataflow requires
+(runner V2)[https://cloud.google.com/dataflow/docs/guides/deploying-a-pipeline#dataflow-runner-v2]
+and Beam 2.22.0 or later. See
+[here](https://beam.apache.org/documentation/runners/dataflow/) for 
+instructions for setting up Dataflow.
+
+Setup a virtual environment for running Beam Python programs. See 
+[here](https://beam.apache.org/get-started/quickstart-py/) for prerequisites. 
+Dataflow requires the `gcp` tag. Assuming your work directory to be
+`/path/to/work`.
+
+```sh
+> export WORK_DIRECTORY=/path/to/work
+> cd $WORK_DIRECTORY
+> mkdir kafka_env
+> virtualenv kafka_env
+> . kafka_env/bin/activate
+> pip install -e .[gcp]
+```
+
+Run the Beam pipeline. You can either use the default Kafka topic name or 
+specify a Kafka topic name. Following command assumes Dataflow. See 
+[here](https://beam.apache.org/get-started/quickstart-py/) for instructions on 
+running Beam Python programs on other runners. Note that you have to fill in 
+information mentioned within angle brackets (`<` and `>`).
+
+Note that this exemple is not available in Beam versions before 2.24.0 hence
+you'll have to either get the example program from Beam or follow steps
+provided in the section *Running the Example from a Beam Git Clone*.
+
+```sh
+>  python -m apache_beam.examples.kafkataxi.kafka_taxi --runner DataflowRunner --temp_location <GCS temp location> --project <GCP project>  --region <region> \
```

Comment:
Can we get rid of the `>` from here as well?

I would highly suggest breaking this command into separate lines for readability.

It would also be a good idea to set variables beforehand so that once they set those variables, they can copy-paste the rest of the command.

I would link to the [Dataflow regional endpoints docs](https://cloud.google.com/dataflow/docs/concepts/regional-endpoints) for more information about the `--region`.

```sh
PROJECT="$(gcloud config get-value project)"
TEMP_LOCATION="gs://MY-BUCKET/temp"
REGION="us-central1"
JOB_NAME="kafka_taxi-`date +%Y%m%d-%H%M%S`"
BOOTSTRAP_SERVER="123.45.67.89:123"

python -m apache_beam.examples.kafkataxi.kafka_taxi \
  --runner "DataflowRunner" \
  --project "$PROJECT" \
  --temp_location "$TEMP_LOCATION" \
  --region "$REGION" \
  --num_workers 1 \
  --job_name "$JOB_NAME" \
  --bootstrap_servers "$BOOTSTRAP_SERVER" \
  --experiments "use_runner_v2"
```